lib/matplotlib/testing/nose/plugins/conversion_cache.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from pprint import pprint
+from nose.plugins import Plugin
+import os
+
+from ... import conversion_cache as ccache
+
+
+class ConversionCache(Plugin):
```

Comment:
Looks like you have two classes both named ConversionCache.  While technically OK I guess it'd be more legible to rename e.g. this one as ConversionCachePlugin (or similar).